src/components/Calculator.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Kindly remove the eslint-disabling commands from all your files and try to fix them🙏 , some of them are so important to keep your code readable & clean 🙏